app/utils/ExtractMeta.js

```diff
@@ -57,6 +58,26 @@ export default function extractMeta(chain_data, rp) {
         } else {
             addSiteMeta(metas);
         }
+    } else if (rp.accountname) { // user profile root
+        const account = chain_data.accounts[rp.accountname];
+        let {name, about, profile_image} = normalizeProfile(account);
+        if(name == null) name = account.name;
+        if(about == null) about = " Steemit";
+        if(profile_image == null) profile_image = 'https://steemit.com/images/steemit-twshare.png';
+        // Set profile tags
+        const title = `@${account.name}`;
+        const desc  = `The latest posts from ${name}. Follow me at @${account.name}. ${about}`;
+        const image = profile_image;
+
+        // Standard meta
+        metas.push({name: 'description', content: desc.substring(0,170)});
```

Comment:
Since the "about" field already has an enforced max size, I think we can get rid of the forced substring. We wouldn't be hitting the 170 char limit too often.